### ISSUE TITLE
Swift scripting codegen error: "Error: Cannot find module './errors/pretty-print'"

### ISSUE THREAD

**Issue by nateirwin** [apollographql/apollo-ios#1331]
macOS 10.15.6
Xcode 11.6

We're using Swift scripting for Codegen in our project. It was working fine until recently. The only change that occurred (that I can think of) is macOS and Xcode updated. I'm getting the following error:

```
[DEBUG - ApolloCodegenLib:CLIDownloader.swift:43] - Zip file with the CLI is already downloaded!
SHASUM of downloaded file: c2b1215eb8e82ec9d777f4b1590ed0f60960a23badadd889e4d129eb08866f14
[DEBUG - ApolloCodegenLib:CLIExtractor.swift:57] - Binary already extracted!
[ERROR - ApolloCodegenLib:Basher.swift:62] - internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module './errors/pretty-print'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/nateirwin/Development/trailheadlabs/outerspatial-ios/Codegen/ApolloCLI/apollo/node_modules/@oclif/errors/lib/handle.js:6:24)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

Program ended with exit code: 1
```

**Comment by designatednerd** [comment]
That's very odd, that shouldn't have changed anything in the path - I did not have any migration issues when I migrated my machine to those versions. 

What I'd suggest is deleting the extracted files from the `tar.gz` file and letting it re-extract. I have no idea what could have changed in there, but re-extracting from the zip file is a good way to make sure you've got everything. Let me know if this works.
